lib/internal/Magento/Framework/Mview/View/Subscription.php

```diff
@@ -193,11 +193,14 @@ protected function getLinkedViews()
      * Build trigger statement for INSERT, UPDATE, DELETE events
      *
      * @param string $event
-     * @param \Magento\Framework\Mview\View\ChangelogInterface $changelog
+     * @param ViewInterface $view
      * @return string
      */
-    protected function buildStatement($event, $changelog)
+    protected function buildStatement($event, $view)
```

Comment:
Do you believe it's backward compatible change of changing argument type for existing protected method?

Reply:
As we discussed with @gabrieldagama  - this changes is allowed since this class isn't marked with `@api` tag.
Could you define argument & return types here?